Makefile

```diff
@@ -65,34 +63,34 @@ get_deps:
 
 ensure_tools:
 	go get -u -v $(GOTOOLS)
-	@gometalinter.v2 --install
+	@ gometalinter.v2 --install
 
 get_vendor_deps: ensure_tools
-	@rm -rf vendor/
-	@echo "--> Running glide install"
+	@ rm -rf vendor/
+	@ echo "--> Running glide install"
 	@ glide install
 
-metalinter:
+metalinter_all:
 	protoc $(INCLUDE) --lint_out=. types/*.proto
 	gometalinter.v2 --vendor --deadline=600s --enable-all --disable=lll ./...
 
 metalinter_test:
-	#protoc $(INCLUDE) --lint_out=. types/*.proto
```

Comment:
why remove proto linter???

Reply:
Is proto linter fast enough to run on ci?

Reply:
we don't need it, and it's a vulnerability to depend on it.

Reply:
like, all the stuff in GOTOOLS pretty much needs to be installed for anyone to start building, but that means installing something not used often... if that repo gets hacked, we've hacked all devs trying out ABCI.

Reply:
oh hmm... is --lint_out a default proto feature?  I removed a third party dependency from GOTOOLS and I thought --lint_out depended on that.

Reply:
Oops I forgot to remove https://github.com/ckaznocha/protoc-gen-lint as well.  That was the point.